### DISCUSSION TITLE
Normalization/Standardization input data

### DISCUSSION THREAD

**Discussion by krishdotn1** [AI4Finance-Foundation/FinRL#1180] [Ideas]
Is it not a good idea to normalize/standardize the observation space used in the environment for training our model?
 The observation space often consists of a much higher number of features [i.e 100000, 180,181.5,180.2......], which can lead the model to be mistrained. 

Additionally, computing with higher numbers requires more computational power

@zhumingpassional

Thanks

**Comment by zhumingpassional** [comment]
pls submit issues on github

Wish you all my best.
Sincerely,
Ming Zhu.


Krish Nandan Das ***@***.***> 于2024年3月6日周三 18:41写道：
